scripts/check_urls.lua

```diff
@@ -0,0 +1,93 @@
+local ts = vim.treesitter
+local async = require('vim._async')
+
+local query = ts.query.parse('vimdoc', '(url) @url')
+
+local function curl(url, cb)
+  local cmd = {
+    'curl',
+    '--silent',
+    '-L',
+    '--max-time',
+    '5',
+    '--fail',
+    '--output',
+    '/dev/null',
+    url,
+  }
+  vim.system(cmd, cb)
+end
+local request = async.wrap(2, curl)
```

Comment:
`vim.system` is already async so don't need `vim.async`

Reply:
I'll rewrite it with vim.async removed and will report back